ckanext/xloader/config_declaration.yaml

```diff
@@ -10,12 +10,12 @@ groups:
             Postgresql database.
         validators: not_missing
         required: true
-      - key: ckanext.xloader.api_token
-        example: eyJ0eXAiOiJKV1QiLCJh.eyJqdGkiOiJ0M2VNUFlQWFg0VU.8QgV8em4RA
+      - key: ckanext.xloader.user
+        example: xloader_system
         description: |
-            Uses a specific API token for the xloader_submit action instead of the
-            apikey of the site_user. Will be mandatory when dropping support for
-            CKAN 2.9.
+            The username of a sysadmin that will be executing the actions of the
+            xloader service. This user should also have an API token to download
```

Comment:
why does it matter if the user has a token? Tokens in an account can't be read and used

Reply:
Oops! Was going off of the sysadmin object, but yeah I see that now. Brought back the old `ckanext.xloader.api_token` option with revised description.